keyboards/lck75/lck75.c

```diff
@@ -14,4 +14,140 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-#include "lck75.h"
\ No newline at end of file
+
+#include "lck75.h"
+
+__attribute__((weak)) void encoder_update_user(uint8_t index, bool clockwise) {
+    if (index == 0) {
+        if (clockwise) {
+            tap_code(KC_VOLU);
+        } else {
+            tap_code(KC_VOLD);
+        }
+    }
+}
+
+extern uint8_t is_master;
+
+#define IDLE_FRAMES 5
+#define IDLE_SPEED 30
+#define TAP_FRAMES 2
+#define TAP_SPEED 40
+#define ANIM_FRAME_DURATION 200
+#define ANIM_SIZE 512
+
+bool gui_on = true;
+uint32_t anim_timer = 0;
+uint32_t anim_sleep = 0;
+uint8_t current_idle_frame = 0;
+uint8_t current_tap_frame = 0;
+
+static void render_anim(void) {
```

Comment:
I would say move all of this into the oled_task_user function.

Reply:
updated the oled code